data/tech/aria/aria-colcount_attribute.json

```diff
@@ -9,6 +9,46 @@
       "url": "https://www.w3.org/TR/wai-aria-1.1/#aria-colcount"
     }
   ],
-  "assertions": [],
-  "date_updated": "2019-01-06"
+  "assertions": [
+    {
+      "id": "convey_column_count_all_cols",
+      "title": "convey the total column count when all columns are visible",
+      "rationale": "The user needs to know how many columns are in the whole grid / table",
+      "strength": {
+        "sr": "MUST",
+        "vc": "NA"
+      },
+      "css_target": "table[aria-colcount]",
+      "operation_modes": [
+        "sr/reading"
+      ]
+    },
+    {
+      "id": "convey_column_count_hidden_cols",
+      "title": "convey the total column count when some columns are hidden",
+      "rationale": "The user needs to know how many columns are in the whole grid / table when some columns are hidden",
+      "strength": {
+        "sr": "MUST",
+        "vc": "NA"
+      },
+      "css_target": "table[aria-colcount]",
+      "operation_modes": [
+        "sr/reading"
+      ]
+    },
+    {
+      "id": "do_not_calculate_unknown_column_count",
+      "title": "convey a column count when total column count is unknown",
+      "rationale": "Screen readers should not attempt to calculate the column count when it is unknown",
```

Comment:
Also wondering if we should call out the "-1" value here at all (similar to my comment for aria-rowcount)

Reply:
Added verbatim note from ARIA spec. ✔️